client/signup/style.scss

```diff
@@ -294,3 +294,45 @@ body.is-section-signup .layout.gravatar .formatted-header {
 		}
 	}
 }
+
+// Background color override for new flow
```

Comment:
Kept this separate for now, as opposed to adding a bunch of `:not()`s that we'll have to track down later